src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -107,7 +107,7 @@ public static AttributeCollection FromExisting(AttributeCollection existing, par
             // Now, if we collapsed some attributes, create a new array.
             //
 
-            Attribute[] attributes = null;
+            Attribute[] attributes = Array.Empty<Attribute>();
```

Comment:
Does this need to be assigned at all?  It's set in both the subsequent if/else branch, so it should be definitely assigned without needing this initializer.